portfolio/src/main/webapp/index.html

```diff
@@ -4,11 +4,53 @@
     <meta charset="UTF-8">
     <title>Anesia's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script src ="https://www.gstatic.com/charts/loader.js"></script>
     <script src="script.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
```

Comment:
Just curious why these style definitions are here instead of style.css?  If there are reasons, it would be nice to add them as comments.  css is something that can get very hard to keep track of, especially if it is scattered around in different files.